addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -27,6 +27,12 @@
                   class="badge bg-success">
                     <i class="fa fa-check me-1"/>Solved
             </span>
+            <span t-if="question.state=='pending'" 
+                  title="Pending"  
```

Comment:
I see what you did with the alignment, but I think we should instead fix the span above with this one to do as above.